newsroom/default_settings.py

```diff
@@ -97,6 +100,8 @@
 WEBPACK_ASSETS_URL = os.environ.get('ASSETS_URL', '/static/dist/')
 WEBPACK_SERVER_URL = os.environ.get('WEBPACK_SERVER_URL', 'http://localhost:8080/')
 
+SERVER_DOMAIN = 'localhost'
```

Comment:
why is this needed?

Reply:
Superdesk-core was complaining about it when I used the function `get_recurring-events` so I put this in to pass that